bedrock/newsletter/templates/newsletter/includes/multi-newsletter-form.html

```diff
@@ -0,0 +1,145 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+ {% from "newsletter/includes/macros.html" import email_form_thankyou with context %}
+
+ {% if not success %}
+   <form id="newsletter-form" class="mzp-c-newsletter-form" action="{{ url('newsletter.subscribe') }}" method="post">
+     <input type="hidden" name="source_url" value="{{ request.build_absolute_uri() }}">
+     <!-- todo: figure out if we want to keep the dropdown -->
```

Comment:
removed hidden newsletters input (ids selected by checkbox)
added dummy div to avoid breaking selector in protocol JS